### ISSUE TITLE
Pre-set import settings can't be deleted

### ISSUE THREAD

**Issue by shinwa** [salesagility/SuiteCRM#3736]
#### Issue

I've tested on the demo site too, for both account & opportunity modules, I think this may affect all modules that have import pre-set settings.

#### Expected Behavior
After I have saved the pre-set, I went back to import wizard page and hit delete button, then click OK. The pre-set should be gone.

#### Actual Behavior
But If i refresh the page, the pre-set come back again. You can add new pre-sets, but can't delete them.

#### Possible Fix
Don't know why. Maybe some permission issue?

#### Steps to Reproduce
1. Import some records and save as pre-set
2. Go back to import wizard page
3. Delete the pre-set
4. Refresh the page

#### Context
I just want to keep the page clean and delete unwanted/test pre-set to avoid some trouble when importing records.
It doesn't effect the use, maybe medium to low priority.

#### Your Environment

* SuiteCRM Version used: 7.8.3, and the SuiteCRM demo site also have this bug
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)):
* Environment name and version (e.g. MySQL, PHP 7): PHP 7, MariaDB, Nginx.
* Operating System and version (e.g Ubuntu 16.04): CentOS 7

**Comment by ChangezKhan** [comment]
@shinwa are you getting any error in browser console?

**Comment by shinwa** [comment]
Nope, I've tried both IE and Chrome, only a confirm dialog pop-up and ask me to confirm the delete, then i clicked "OK", the pre-set will be gone until i refresh the page.

**Comment by johnM2401** [comment]
Hey!
Thank you for raising this!

I'm going to close this off for now as the linked PR to resolve this has been merged as of 7.10.x

However, if you are still seeing this issue in the latest releases of SuiteCRM, please don't hesitate to get in touch.

Thank you for your contributions to the project!
